.github/pull_request_template.md

```diff
@@ -1,2 +1,3 @@
 ```release-note
+NONE
```

Comment:
I didn't do this before because I was worried that people wouldn't think "should I write a release note?" any more